ui/ui_term.c

```diff
@@ -193,15 +193,14 @@ uint32_t ui_term_color_text_background(uint32_t rgb_text, uint32_t rgb_backgroun
         default:
             break;
     }
-
     return 0;
 }
 
-uint32_t ui_term_color_text_background_buf(char* buf, uint32_t rgb_text, uint32_t rgb_background) {
+uint32_t ui_term_color_text_background_buf(char* buf, size_t len, uint32_t rgb_text, uint32_t rgb_background) {

```

Comment:
Please change new parameter `len` to `bufLen`.

I personally prefer more verbosity (`buf_cch` for count of characters), but `bufLen` is OK in this instance since each element is one byte in size.